packages/loopring_v3/contracts/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -60,16 +48,10 @@ library ExchangeBlocks
         uint8 previousMakerFeeBips
     );
 
-    event BlockFeeWithdrawn(
-        uint    indexed blockIdx,
-        uint            amountRewarded,
-        uint            amountFined
-    );
-
     function submitBlocks(
-        ExchangeData.State storage S,
-        ExchangeData.Block[] memory blocks,
-        address payable feeRecipient
+        ExchangeData.State   storage S,
+        ExchangeData.Block[] memory  blocks,
+        address              payable feeRecipient
         )
         public
     {
```

Comment:
public --> external ?

Reply:
Internal compiler error. Structs + `calldata` in functions are a strange beast, I'm not sure what is allowed and what isn't.